.vscode/launch.json

```diff
@@ -7,7 +7,7 @@
             "type": "extensionHost",
             "request": "launch",
             "runtimeExecutable": "${execPath}",
-            "args": ["--extensionDevelopmentPath=${workspaceRoot}" ],
+            "args": ["--extensionDevelopmentPath=${workspaceRoot}", "/home/jholzl/Projects/lean-scratch/coinduction.lean"],
```

Comment:
You accidentally committed a local path here.  I have never needed to add anything here, F5 always re-opens the last folder (that I opened in F5).